docs/pages/versions/unversioned/sdk/bar-code-scanner.md

```diff
@@ -67,10 +67,12 @@ export default function App() {
   const [scanned, setScanned] = useState(false);
 
   useEffect(() => {
-    (async () => {
+    const getBarCodeScannerPermissions = async () => {
       const { status } = await BarCodeScanner.requestPermissionsAsync();
-      setHasPermission(status === 'granted');
-    })();
+      setHasPermission(status === 'granted' ? true : false);
```

Comment:
```suggestion
      setHasPermission(status === 'granted');
```

Hello @MaeWolff, thank you for adding name to the permission effect function for the clarity. 👍 

Before we merge it I have a question: What's the reasoning behind updating the line above? Compare like that [can only yield a Boolean result](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality).

Reply:
Hi @Simek ! 

There's not really reasoning, I thought it was more readable at first sight. I can remove my modification if you want

Reply:
Let's revert to the previous form than, and we can ship it. 🙂

Reply:
[4562eaa](https://github.com/expo/expo/pull/17747/commits/4562eaacdf29f765a8003f8f39f99411c01bf21e) :)